main.nf

```diff
@@ -142,13 +142,36 @@ process chromSizes {
 } 
 
 
+/*
+ * Uncompress VCF file
+ */
+process uncompressInputVCF {
+
+    input:
+        path "source.vcf" from params.vcffile
+
+    output:
+        path "uncompressed.vcf" into vcf_file
+
+    script:
+        if ( file(params.vcffile).getExtension() == 'gz' )
```

Comment:
```suggestion
        if( file(params.vcffile).getExtension() == 'gz' )
```

While this is indeed not how it's done in Python, and I even agree with @apriltuesday that this is perhaps not the most elegant way to my eyes, I think we should still follow the conventions of the language as they're laid out in the docs/style guide.